pkg/util/file_test.go

```diff
@@ -0,0 +1,29 @@
+package util
+
+import (
+	"errors"
+	"io/fs"
+	"os"
+	"testing"
+
+	"github.com/awslabs/aws-s3-csi-driver/pkg/util/testutil/assert"
+)
+
+func TestFileGroupIDReturnsProcessGid(t *testing.T) {
+	tmpFile, err := os.CreateTemp("", "testfile")
```

Comment:
FYI, you can use [`t.TempDir()`](https://pkg.go.dev/testing#T.TempDir) to get a unique temporary directory, and after the test case it will be cleaned up automatically.

Reply:
Thanks, I didn't know.